### ISSUE TITLE
Add LG backup solution

### ISSUE THREAD

**Issue by steadfasterX** [Carbon-Fusion/build_fwul#41]
The following will work for any LG as long as supported by the lglaf tool: 

https://forum.xda-developers.com/g4/help/tutorial-backup-download-mode-t3659223

The idea is to use yad as the GUI and before starting the backup asking for USB stick to get the output dir automatically.

**Comment by steadfasterX** [comment]
superseded by SALT #59
